python/lsst/meas/base/dataIds.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
```

Comment:
Is this new file really the best location for this code? It rankles a bit since it's cluttering the (already disorganized) `lsst.meas.base` namespace with something that's by no stretch of the imagination a "core measurement algorithm" (as the repository description claims).

As far as I can tell, this is only used by `ForcedPhotCcdTask`. Do you expect it to have wider applicability? If not, can you just fold it into `forcedPhotCcd.py`?